src/utils/creds.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs')
 var netrc = require('netrc')
 var os = require('os')
 
-var host = 'api.storyblok.com'
+var host = 'api-us.storyblok.com'
```

Comment:
This host variable is used in login in this file I think, I believe it doesn't need to be changed, what do you think ?